### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Its-Alex**
Some dependencies are outdated, in this PR I update them.

I had some difficulties with `react-bootstrap`, @stephane-klein what do you think about the website now?

### CODE REVIEW

**Comment by stephane-klein** [comment]
@Its-Alex **issue:** when I execute:

```
$ cd frontend
$ yarn install
$ yarn run start
```

I have this error message in my browser console log:

```
Uncaught TypeError: GetByPathWithDefault(...) is undefined
    Home App.js:80
    React 12
    unstable_runWithPriority scheduler.development.js:468
    React 5
    _callee2$ index.js:210
    tryCatch runtime.js:63
    invoke runtime.js:293
    defineIteratorMethods runtime.js:118
    Babel 2
App.js:80
    Home App.js:80
    React 11
    performSyncWorkOnRoot self-hosted:1220
    flushSyncCallbackQueueImpl React
    unstable_runWithPriority scheduler.development.js:468
    React 5
    dispatchAction self-hosted:1277
    _callee2$ index.js:210
    tryCatch runtime.js:63
    invoke runtime.js:293
    defineIteratorMethods runtime.js:118
    Babel 2
```

**Comment by Its-Alex** [comment]
@stephane-klein https://github.com/spacefill/gibbon-mail/pull/18#issuecomment-816262933 Is fixed :heavy_check_mark:

File: frontend/public/index.html [hunk 1]
```diff
@@ -6,14 +6,10 @@
...
-      crossorigin="anonymous"/>
-
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css"
-      />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css"
+      integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+      crossorigin="anonymous"
+    />
```
Comment (stephane-klein) [inline frontend/public/index.html:12]: @Its-Alex **question:** why `bootstrap-theme.min.css` is removed?

**Comment by stephane-klein** [comment]
@Its-Alex Tested and merged, thanks 👌

**Pull request merged**